test/unit/IntentReader.test.js

```diff
@@ -64,6 +64,31 @@ jest.mock('@firebolt-js/sdk/dist/lib/Transport/index.mjs', () => {
   };
 });
 
+jest.mock('@firebolt-js/sdk', () => {
+  return {
+    Accessibility: {},
+    Account: {},
+    Advertising: {},
+    Authentication: {},
+    Device: {},
+    Discovery: {},
+    Keyboard: {},
+    Lifecycle: {
+      ready: () => {},
+      state: () => {
+        return 'initializing'; // dummy state value.
+      }, // returning a Lifecycle.state object
+      close: () => {},
+      finish: () => {},
+    },
+    Localization: {},
+    Metrics: {},
+    Profile: {},
+    Parameters: {},
+    SecondScreen: {},
+  };
+});
+
```

Comment:
what is this mock for ? its not used anywhere . there are linting issues too

Reply:
unit testcases were failing without this mock after adding the new handler.

Reply:
linting issues are fixed